docs/advanced-entity-analytics/behavioral-detection-use-cases.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[behavioral-detection-use-cases]]
+= Behavioral detection use cases
+
+Behavioral use cases detect potential insider and external threats based on observed user actions and host activities. These use cases employ a threat-centric approach to flag deviations or suspicious activities by analyzing patterns, anomalies, and context enrichment. 
```

Comment:
```suggestion
Behavioral detection identifies potential internal and external threats based on user and host activity. It employs a threat-centric approach to flag deviations from the norm and suspicious activity by analyzing patterns, anomalies, and context enrichment. 
```
I'm wondering about the capitalization / proper name for this set of features, since on the Advanced Behavioral Detections page, you capitalized "Advanced Behavioral Detections", but that full name isn't used here. Not sure what's correct for this feature but we should be consistent. In other words I'm wondering if this paragraph should start with "Advanced Behavioral Detections identify..."?

Reply:
The full feature hierarchy is:
- Advanced Entity Analytics
  - Entity Risk Scoring
  - Advanced Behavioral Detections
    - Anomaly detection
    - Behavioral detection use cases
    
Essentially, anomaly detection and behavioral detection use cases are two capabilities of the Advanced Behavioral Detections feature. In a previous PR where Entity Risk Scoring was first introduced, it was agreed to capitalize it as a feature (similar to how we capitalize AI Assistant, for example). I capitalized Advanced Behavioral Detections since it's at the same hierarchy level as Entity Risk Scoring. I'm not sure if we classify anomaly detection and behavioral detection use cases as features – @SourinPaul, @jmikell821 what are your thoughts?

Reply:
I am leaning towards sentence case for `Advanced behavioral detections` for consistency, unless @SourinPaul has a business need otherwise.

Reply:
In that case, should we also lowercase `Entity risk scoring`?

Reply:
To close the loop on this, per the slack convo, we're lowercasing both `advanced behavioral detections` and `entity risk scoring`.